### ISSUE TITLE
Incorrect/misleading descriptions for operation codes SAVE and LOAD

### ISSUE THREAD

**Issue by mikeweilgart** [PleasingFungus/Silicon-Zeroes#131]
On the level "Basic CPU," the "SAVE" operation shows up.

Mousing over a SAVE instruction in memory, for instance "SAVE 6 3 -", shows:

> Set the memory slot specified by register 3 to the value from register 6.

Mousing over the SAVE op code (in the instruction decoder) shows:

> Store the value from the source register into the target memory slot.

This is incorrect.  The contents of register 6 (the source register) should NOT be stored into memory slot 3, even though the target of this instruction is 3.  Instead, the value from the target *register* (register 3) specifies what memory slot to write to.  Saying "target memory slot" is wrong and misleading (and doubly so since previously encountered operations such as "MSET" and "ADDM" *do* directly name a memory slot in the instruction).

My suggested rewording:

> Store the value from the source register into the memory slot specified in the target register.

----

Similarly, the LOAD operation shows up in the level "Load CPU."

Mousing over the instruction "LOAD - 4 5" shows:

> Set register 5 to the value from the memory slot specified by register 4.

However, mousing over the op code "LOAD" in the instruction decoder shows:

> Store the value from the target memory slot into the destination register.

Again, it's not a target memory slot.  It's a memory slot specified by the target register.

Suggested rewording:

> Store the value from the memory slot specified by the target register into the destination register.

------

Actually, even with my rewordings, it might be preferable to have the opcode wording and the instruction wording line up.

Instead of:

> Save: Set the memory slot specified by register 3 to the value from register 6.

> SAVE: Store the value from the source register into the memory slot specified in the target register.

Change the latter to simply match the wording for a specific instruction:

> SAVE: Set the memory slot specified by the target register to the value from the source register.

Likewise for Load:

> Load: Set register 5 to the value from the memory slot specified by register 4.

> LOAD: Store the value from the memory slot specified by the target register into the destination register.

> LOAD: Set the destination register to the value from the memory slot specified by the target register.

That would make everything much clearer regarding these instructions.

-----

On a related note but much more minor, the GRTR opcode explanation would benefit from changing "destination" to say "destination memory slot."

> GRTR: If the source is greater than the target, store 1 in the destination **memory slot**. Otherwise, store 0.

**Comment by mikeweilgart** [comment]
(And as a side note, since I just now read through the "History" page in "Extras," I think you'd like to know that my six-year-old son is also playing this game and loving it.  He's got two levels left to the end of the second board.  So your labor of love definitely succeeded!)  :)

**Comment by PleasingFungus** [comment]
I agree this is poorly phrased! I think I went with the current tooltip descriptions because I was worried that the more precise wording would be too long, but I agree it's a bit misleading as is. I like your more consistent approach.

Unfortunately, I don't currently have access to a Windows development environment, so it'll be a while until I can fix any of this. I'm noting it down, though!

> I think you'd like to know that my six-year-old son is also playing this game and loving it...

Excellent. Thank you very much for letting me know! Brings a smile to my face :)
